src/rdkafka_mock.c

```diff
@@ -1765,6 +1765,7 @@ void rd_kafka_mock_push_request_errors(rd_kafka_mock_cluster_t *mcluster,
                 errors[i] = va_arg(ap, rd_kafka_resp_err_t);
 
         rd_kafka_mock_push_request_errors_array(mcluster, ApiKey, cnt, errors);
+        va_end(ap);
```

Comment:
nit: Move this to the line after the for-loop

Reply:
Done